lib/cache/senders/impressions_sender.rb

```diff
@@ -0,0 +1,59 @@
+module SplitIoClient
+  module Cache
+    module Senders
+      class ImpressionsSender
+        def initialize(impressions_repository, config, api_key)
+          @impressions_repository = impressions_repository
+          @config = config
+          @api_key = api_key
+        end
+
+        def call
+          # Disable impressions if @config.impressions_queue_size == -1
+          if @config.impressions_queue_size < 0
```

Comment:
do we have a unit test that post impressions when it is disabled to make sure it doesn't blow?